invokeai/backend/ldm/models/model_manager.py

```diff
@@ -370,125 +353,6 @@ def _load_model(self, model_name: str):
             )
         return model, width, height, model_hash
 
-    def _load_ckpt_model(self, model_name, mconfig):
```

Comment:
Do we still support ckpt models and just load them through a different path now?

Reply:
We load .ckpt and .safetensors models by converting them into `diffusers` in RAM. This works 99% of the time.